extension_points/configrepo_extension.md

```diff
@@ -0,0 +1,84 @@
+# Configuration repository Extension
+
+Go supports writing configuration plugins starting `16.7`.
+
+It is a feature which allows you to move pipeline configurations out of GoCD and its cruise-config.xml file into one or more source-control repositories (e.g git), so that you can modify them externally. Such modifications will be seen by a periodic poller in the GoCD server and it will [merge](#merging-configurations) those pipeline configurations into the pipelines it finds in the main configuration XML file. You can see a quick preview video of this feature [here](https://www.youtube.com/watch?v=1AfBxCWRqD8).
```

Comment:
(e.g git) to (e.g. git)
I think it's ok to not have the link to the video the description is clear. The video is long and is titled as "Preview"